tests/gumtest.c

```diff
@@ -269,6 +273,7 @@ main (gint argc, gchar * argv[])
   {
     GumScriptBackend * duk_backend, * v8_backend;
 
+# ifdef HAVE_FRIDA_GLIB
```

Comment:
Should be moved up and merged with the `#if defined (HAVE_GUMJS)` instead, i.e.:

```C
#if defined (HAVE_GUMJS) && defined (HAVE_FRIDA_GLIB)
```